hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -72,6 +72,15 @@ public class HoodieCommonConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Turn on compression for BITCASK disk map used by the External Spillable Map");
 
+  public static final ConfigProperty<String> HOODIE_FS_ATOMIC_CREATION_SUPPORT = ConfigProperty
```

Comment:
after introducing this config, i think we do not need any changes on StorageSchemas

Reply:
IIUC, you mean don't support any FS in FileLock as default?

Reply:
yes, since we introduce this config, I think we don't need any modifies on StorageSchemas according to the above comments (users would use this config to control the FS atomic creation instead of modifying source code)